tests/Jellyfin.Naming.Tests/TV/EpisodePathParserTest.cs

```diff
@@ -23,7 +23,7 @@ public void ParseEpisodesCorrectly(string path, string name, int season, int epi
             Assert.Equal(episode, res.EpisodeNumber);
 
             // testing other paths delimeter
-            var res2 = p.Parse(path.Replace('/', '\\'), false);
+            var res2 = p.Parse(path.Replace('/', '/'), false);
```

Comment:
I am not sure why this was changed.
The idea behind the original replacement was to include windows and unix directory delimiter in the test cases without having to write both. To me, both regexp seem to have to match the path.
Similar replacement was also done below.

Reply:
I did a find+replace for backslashes as they fail when tested on linux.